drivers/include/periph/spi.h

```diff
@@ -172,11 +151,9 @@ int spi_release(spi_t dev);
  * @param[in] dev       SPI device to use
  * @param[in] out       Byte to send out, set NULL if only receiving
  * @param[out] in       Byte to read, set NULL if only sending
- *
- * @return              Number of bytes that were transfered
- * @return              -1 on error
  */
-int spi_transfer_byte(spi_t dev, char out, char *in);
+void spi_transfer_byte(spi_t bus, spi_cs_t cs, bool cont,
```

Comment:
that seemed to have fallen of the wagon, somewhere between the branches, will fix.